src/core/internal/string.d

```diff
@@ -56,7 +56,7 @@ char[] unsignedToTempString()(ulong value, return scope char[] buf, uint radix =
 private struct TempStringNoAlloc
 {
     // need to handle 65 bytes for radix of 2 with negative sign.
-    private char[65] _buf;
+    private char[65] _buf = 0;
```

Comment:
Maybe use `= void` here, too. Since recently, it has the same effect as zero-initialization regarding the .init-value handling, but could allow some optiizations. `= void` also hints that there is no enforced zero-termination.

Reply:
Done.